Pspcl/Pspcl.Web/Views/Report/StockInReport.cshtml

```diff
@@ -47,12 +70,13 @@
 
         </table>
 
-
+        <div id="paginate"></div>
 
                
 
     </div>
-    
-    
+
+    @Html.Partial("_ModalPartial")
```

Comment:
What is the purpose of this view placing ? Name should be meaningful, with which we can understand the purpose.

Reply:
actually this is the modal that is used in two pages so I have made it partial and  we can place the common code in this partial view for further use.

Reply:
What purpose actually it poses?

Reply:
It is the modal that is showed in case of validations in both stock in and stock out report ..so to remove code redundancy I have made it partial view or otherwise I have to include this modal in both pages separately.

Reply:
we use same type of modals in different pages . So , to optimize the code we can remove that similar modals in all pages and include it in one common partial view that can use wherever we want.

Reply:
I can remove it and include this modal in both pages or I can change the name of this partial view as it serve the purpose of common html code.